src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -280,6 +278,24 @@ public final void setDefaultHeaders(final List<Header> headers) {
     }
 
     /**
+     * Print a curated set of important HTSJDK default settings to the logger
+     */
+    protected void printHTSJDKDefaults() {
```

Comment:
That's a good idea @magicDGS -- glad you're around to keep GATK4 usable as a library :) I'll rename to `printLibraryDefaults()`